Formula/yaws.rb

```diff
@@ -24,17 +24,28 @@ class Yaws < Formula
   depends_on "libtool" => :build
   depends_on "erlang"
 
+  on_linux do
+    depends_on "linux-pam"
+  end
+
   # the default config expects these folders to exist
   skip_clean "var/log/yaws"
   skip_clean "lib/yaws/examples/ebin"
   skip_clean "lib/yaws/examples/include"
 
   def install
+    # Ensure pam headers are found on Xcode-only installs
+    extra_args = %W[
+      --with-extrainclude=#{MacOS.sdk_path}/usr/include/security
+      SED=/usr/bin/sed
```

Comment:
```suggestion
```
We can remove this, since this was probably there to get around the `sed` shim.

Reply:
Good catch, @carlocab. It builds and tests OK locally on Intel Big Sur, but since it technically changes the macOS build process, does that mean a macOS-only revision bump is now required?

Reply:
Not necessary I think. This line probably didn't do much before; I don't think changing it is worth forcing a reinstall.